packages/concerto-core/api.txt

```diff
@@ -237,13 +240,14 @@ class ValidatedResource extends Resource {
    + void validate() throws Error
 }
 class ModelLoader {
-   + ModelManager[] loadModelManager(string[],object,boolean?,number?) 
-   + ModelManager[] loadModelManagerFromModelFiles(object[],string[],object,boolean?,number?) 
+   + ModelManager[] loadModelManager(string[],object,boolean?,boolean?,number?) 
+   + ModelManager[] loadModelManagerFromModelFiles(object[],string[],object,boolean?,boolean?,number?) 
```

Comment:
These signatures don't match the JS API. Perhaps it's a bug in the JS/jsdoc parser?

Reply:
Yes - it doesn't handle the nested options syntax correctly. I don't think it matters too much - it is only used to detect breaking API changes.